subsys/bluetooth/services/Kconfig.cts

```diff
@@ -0,0 +1,14 @@
+# Bluetooth GATT Battery service
+
+# Copyright (c) 2024 Croxel Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_CTS
+	bool "GATT Current Time service"
+
+if BT_CTS
+
+config BT_CTS_HELPER_API_ENABLE
+	bool "Helper APIs to encode and decode CTS formatted time"
```

Comment:
I'd drop the `_ENABLE` since it's implicit, and (AFAIK) we don't have a convention of naming boolean Kconfig options this way.